dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Voltage/SecondaryVoltageControlSimp.mo

```diff
@@ -0,0 +1,83 @@
+within Dynawo.Electrical.Controls.Voltage;
+
+/*
+* Copyright (c) 2022, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source suite of simulation tools for power systems.
+*/
+
+model SecondaryVoltageControlSimp "Model for simplified secondary voltage control"
+  import Modelica;
+  import Dynawo.Types;
+
+  //Regulation parameters
+  parameter Types.PerUnit Ck "Sensitivity of voltage to level variations in pu (base UNom)";
+  parameter Types.PerUnit DnsDtm "Level slope limitation in pu/min (base QNomAlt)";
+  parameter Types.Time tau "PI time constant in s";
+  parameter Types.Time tI "Closed loop target time constant in s";
+
+  //Input variables
+  Modelica.Blocks.Interfaces.RealInput UpRefPu(start = UpRef0Pu) "Reference voltage of pilot point in pu (base UNom)" annotation(
+    Placement(visible = true, transformation(origin = {-220, 0}, extent = {{-20, -20}, {20, 20}}, rotation = 0), iconTransformation(origin = {120, -60}, extent = {{20, -20}, {-20, 20}}, rotation = 0)));
+  Modelica.Blocks.Interfaces.RealInput UpPu(start = Up0Pu) "Voltage of pilot point in pu (base UNom)" annotation(
+    Placement(visible = true, transformation(origin = {-220, -60}, extent = {{-20, -20}, {20, 20}}, rotation = 0), iconTransformation(origin = {-120, 0}, extent = {{-20, -20}, {20, 20}}, rotation = 0)));
+
+  //Output variable
+  Modelica.Blocks.Interfaces.RealOutput level(start = Level0) "Level demand" annotation(
+    Placement(visible = true, transformation(origin = {230, 6}, extent = {{-10, -10}, {10, 10}}, rotation = 0), iconTransformation(origin = {110, 0}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+
+  Modelica.Blocks.Nonlinear.Limiter limiter(uMax = 1) annotation(
+    Placement(visible = true, transformation(origin = {190, 6}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Modelica.Blocks.Math.Add add annotation(
+    Placement(visible = true, transformation(origin = {150, 6}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Modelica.Blocks.Continuous.LimIntegrator limIntegrator(outMax = 1, y_start = Level0) annotation(
+    Placement(visible = true, transformation(origin = {90, 0}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Modelica.Blocks.Nonlinear.Limiter limiter1(uMax = DnsDtm / 60) annotation(
```

Comment:
Would it make sense to have the 60 not hardcoded but being a parameter (that could be equal to 60 by default)?

Reply:
This 60 is to go from minutes to seconds so to me it's hard coded. What I can do is have DnSDtm in another unit to avoid this 60.

Reply:
Ok. We can keep it this way thus.